e2e/libs/keywords/workload_keywords.py

```diff
@@ -38,10 +38,14 @@ def __init__(self):
 
     def init_storageclasses(self):
         create_storageclass('longhorn-test')
+        create_storageclass('longhorn-test-1-replica')
```

Comment:
Eventually we will pass arbitrary parameters to customize storage class to meet test cases requirements, like `fromBackup`, `migratable`, `dataLocality`, `backingImageName` and so on (you can reference https://longhorn.io/docs/1.6.1/references/storage-class-parameters/ to see all parameters). Is it possible to modify the mechanism to dynamically create custom storage class on demand (before you create the PVC) and destroy it when the test case completed?